docs/guides/create_and_upload_dataset.md

```diff
@@ -196,4 +197,43 @@ NAME
 alice/datasets/my-congress-members/1
 ```
 
-Feel free to navigate over to that Dataset in your FloydHub Dashboard to explore the dataset further.
\ No newline at end of file
+Feel free to navigate over to that Dataset in your FloydHub Dashboard to explore the dataset further.
+
+### Job
```

Comment:
Rename this to "promote job output to a dataset" to be more obvious.

Other than downloading data from the internet, users can also use it to save model files etc. So may be include that info as well.